open_api_framework/conf/base.py

```diff
@@ -950,25 +950,47 @@ def init_sentry(before_send: Callable | None = None):
 # ideally we'd use BASE_URI but it'd have to be lazy or cause issues
 CSP_DEFAULT_SRC = [
     "'self'",
-] + config("CSP_EXTRA_DEFAULT_SRC", default=[], split=True)
+] + config(
+    "CSP_EXTRA_DEFAULT_SRC",
+    default=[],
+    split=True,
+    help_text="Extra default source URLs for CSP other than ``self``. "
+    "Used for ``img-src``, ``style-src`` and ``script-src``",
+)
 
 CSP_REPORT_PERCENTAGE = config("CSP_REPORT_PERCENTAGE", 1.0)  # float between 0 and 1
```

Comment:
Let's also add a help text for this one

Reply:
I've changed Default to 0 since there is no `report-uri` Should I just remove the reporting?

Reply:
I think this is fine, the report uri is configurable if it's really needed, so default no reporting seems okay to me